frontend/simple/views/components/Voting/Voting.vue

```diff
@@ -130,36 +153,130 @@ export default {
   },
   data () {
     return {
-      closeProposalState: {
+      ephemeral: {
         state: null,
         countdown: null
       }
     }
   },
   computed: {
+    ctas () {
```

Comment:
Can we rename this to something clearly understandable? I keep forgetting what `ctas` means

Reply:
I worked too much with ad agencies so CallToAction is burnt into my brain :) do you have a better suggestion? like `buttonText` or something alike?

Reply:
I make @hubudibu words my words too. CTA for is as clear as JS or UI. 

It's a really simple term that can suit anything with a unique purpose: Main place for the user to interact with. Call To Action.

And better than that, IMO is that reduces my cognitive effort, as developer, to name one more thing... because everyone knows that _Naming Things is Hard_ `¯\_(ツ)_/¯`